sync-core/src/test/java/com/cloudant/common/CouchTestBase.java

```diff
@@ -57,7 +57,7 @@
             System.getProperty("test.couch.ignore.compaction",Boolean.FALSE.toString()));
 
     public static final Boolean IGNORE_AUTH_HEADERS = Boolean.valueOf(
-            System.getProperty("test.couch.ignore.auth.headers",Boolean.FALSE.toString()));
+            System.getProperty("test.couch.ignore.auth.headers",Boolean.TRUE.toString()));
```

Comment:
I don't think that answers the question without more context. Why did it work before, for example, why have we changed the requirements on the server?